src/logics/Logic.h

```diff
@@ -274,8 +260,8 @@ class Logic {
     PTRef       mkImpl        (PTRef _a, PTRef _b);
     PTRef       mkNot         (PTRef);
     PTRef       mkNot         (vec<PTRef>&);
-    PTRef       mkIte         (vec<PTRef>&);
-    PTRef       mkIte         (PTRef c, PTRef t, PTRef e);// { vec<PTRef> tmp; tmp.push(c); tmp.push(t); tmp.push(e); return mkIte(tmp); }
+    virtual PTRef mkIte         (vec<PTRef>&);
+    virtual PTRef mkIte         (PTRef c, PTRef t, PTRef e);// { vec<PTRef> tmp; tmp.push(c); tmp.push(t); tmp.push(e); return mkIte(tmp); }
```

Comment:
It does not seem that these methods need to be virtual. No logic is overriding them and I don't see a reason why they should, now.